### PR TITLE
BUGFIX: Delete recipe button deletes record from database

### PR DESCRIPTION
**Pull request by Dabble362**
- fix casing error on deleteRecipe method of recipes.js controller
- changed routing method from delete to post
- changed form method from delete to post

### CODE REVIEW

**Review by jduffey — CHANGES_REQUESTED** [review]
Need to approve/merge this PR first: https://github.com/Dabble362/Mazel/pull/13

File: controllers/recipes.js
Comment (jduffey) [inline controllers/recipes.js:202]: I changed up the logging of this block just a bit when testing locally, if you wanna take a look and consider implementing it. I do think at some point we should go through everything and start standardizing how and where things are logged.

```
deleteRecipe: async (req, res) => {
  console.log('deleteRecipe was invoked')
  try {
    const recipeDbId = req.params.id;
    console.log("   Finding recipe id: ", recipeDbId);
    const recipe = await Recipe.findById({ _id: recipeDbId });
    const recipeCloudinaryId = recipe.cloudinaryId;
    console.log("   Found CloudinaryID on recipe: ", recipeCloudinaryId);
    await cloudinary.uploader.destroy(recipeCloudinaryId);
    console.log("   Deleted CloudinaryID image: ", recipeCloudinaryId);
    await recipe.remove({ _id: recipeDbId });
    console.log("   Deleted recipe ID from db: ", recipeDbId);
    console.log("✅ deleteRecipe has completed successfully");
    res.redirect("/profile");
  } catch (err) {
    console.log("An error has occurred while attempting to delete a recipe.");
    console.log(err);
    res.redirect("/profile");
  }
},
```

Comment (jduffey) [reply controllers/recipes.js:202]: I'm just leaving this as a comment and not a "request". I'll leave it up to you if you want to replace it with what I have here. It's not critical to the PR, just enhances the logging a tiny bit.

**Review by jduffey — CHANGES_REQUESTED** [review]
Since we just merged another PR into `main`, make sure to pull and merge `main` into this branch (and push it) before we continue to review/approve this PR.

File: controllers/recipes.js
Comment (Dabble362) [reply controllers/recipes.js]: it was added. The logs are great.

**Review by Dabble362 — COMMENTED** [review]
these are great

File: middleware/cloudinary.js
Comment (jduffey) [inline middleware/cloudinary.js]: This is now duplicated after the merging of `main`. Remove.

**Comment by jduffey** [comment]
Suggest renaming the title of this PR and the title of the commit to something like 

"BUGFIX: Delete recipe button deletes record from database"

I wouldn't worry too much about including any detail beyond that because the code changes speak for themselves as far as _how_ the bugfix was implemented. Also, the re-adding of the dotenv thing should _not_ be mentioned in this PR because that code change is now integrated as part of the previous PR (https://github.com/Dabble362/Mazel/pull/13) and not this one.

After the duplicate line issue gets resolved (I made a separate comment about it) then we should be good for an approval.

File: middleware/cloudinary.js
Comment (jduffey) [reply middleware/cloudinary.js]: @Dabble362

Comment (jduffey) [reply middleware/cloudinary.js]: Resolved. Ignore this.

**Review by jduffey — APPROVED** [review]
:shipit:

**Pull request merged**
